mock/docs/site-defaults.cfg

```diff
@@ -463,6 +463,16 @@
 
 ## Use this to force foreing architecture (requires qemu-user-static)
 # config_opts['forcearch'] = None
+## mapping from target_arch (or forcearch) to arch in /usr/bin/qemu-*-static
+# config_opts['qemu_user_static_mapping'] = {                                                                                                                                              
+#     'aarch64': 'arm',
+#     'i386': 'i386',
+#     'i686': 'i386',
+#     'ppc64': 'ppc64',
+#     'ppc64le': 'ppc64le',
+#     's390x': 's390x',
+#     'x86_64': 'x86_64',
```

Comment:
I think i386/i686/x86_64 should map to `x86`

Reply:
why?

Reply:
Nope. The binary `/usr/bin/qemu-i386-static` exists.

Reply:
Rather than guessing anymore, here's the list of them:

```
ngompa@Hild ~> ls /usr/bin/qemu-* -hal
-rwxr-xr-x. 1 root root 5.9M Oct 17 13:39 /usr/bin/qemu-aarch64_be-static*
-rwxr-xr-x. 1 root root 5.9M Oct 17 13:39 /usr/bin/qemu-aarch64-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-alpha-static*
-rwxr-xr-x. 1 root root 4.6M Oct 17 13:39 /usr/bin/qemu-armeb-static*
-rwxr-xr-x. 1 root root 4.6M Oct 17 13:39 /usr/bin/qemu-arm-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-cris-static*
-rwxr-xr-x. 1 root root 526K Oct 17 13:39 /usr/bin/qemu-ga*
-rwxr-xr-x. 1 root root 5.5M Oct 17 13:39 /usr/bin/qemu-hexagon-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-hppa-static*
-rwxr-xr-x. 1 root root 3.9M Oct 17 13:39 /usr/bin/qemu-i386-static*
-rwxr-xr-x. 1 root root 2.0M Oct 17 13:39 /usr/bin/qemu-img*
-rwxr-xr-x. 1 root root 2.0M Oct 17 13:39 /usr/bin/qemu-io*
lrwxrwxrwx. 1 root root   18 Oct 17 13:35 /usr/bin/qemu-kvm -> qemu-system-x86_64*
-rwxr-xr-x. 1 root root 3.7M Oct 17 13:39 /usr/bin/qemu-m68k-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-microblazeel-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-microblaze-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mips64el-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mips64-static*
-rwxr-xr-x. 1 root root 4.1M Oct 17 13:39 /usr/bin/qemu-mipsel-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mipsn32el-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mipsn32-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mips-static*
-rwxr-xr-x. 1 root root 2.1M Oct 17 13:39 /usr/bin/qemu-nbd*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-nios2-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-or1k-static*
-rwxr-xr-x. 1 root root 4.5M Oct 17 13:39 /usr/bin/qemu-ppc64le-static*
-rwxr-xr-x. 1 root root 4.5M Oct 17 13:39 /usr/bin/qemu-ppc64-static*
-rwxr-xr-x. 1 root root 4.4M Oct 17 13:39 /usr/bin/qemu-ppc-static*
-rwxr-xr-x. 1 root root 915K Oct 17 13:39 /usr/bin/qemu-pr-helper*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-riscv32-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-riscv64-static*
-rwxr-xr-x. 1 root root 3.8M Oct 17 13:39 /usr/bin/qemu-s390x-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-sh4eb-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-sh4-static*
-rwxr-xr-x. 1 root root 3.6M Oct 17 13:39 /usr/bin/qemu-sparc32plus-static*
-rwxr-xr-x. 1 root root 3.6M Oct 17 13:39 /usr/bin/qemu-sparc64-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-sparc-static*
-rwxr-xr-x. 1 root root 2.9M Oct 17 13:39 /usr/bin/qemu-storage-daemon*
-rwxr-xr-x. 1 root root  14M Oct 17 13:39 /usr/bin/qemu-system-i386*
-rwxr-xr-x. 1 root root  14M Oct 17 13:39 /usr/bin/qemu-system-x86_64*
-rwxr-xr-x. 1 root root 3.9M Oct 17 13:39 /usr/bin/qemu-x86_64-static*
-rwxr-xr-x. 1 root root 6.4M Oct 17 13:39 /usr/bin/qemu-xtensaeb-static*
-rwxr-xr-x. 1 root root 6.4M Oct 17 13:39 /usr/bin/qemu-xtensa-static*
```

Reply:
Rather than to continue guessing, here's the complete list:

```
ngompa@Hild ~> ls -hal /usr/bin/qemu-*-static
-rwxr-xr-x. 1 root root 5.9M Oct 17 13:39 /usr/bin/qemu-aarch64_be-static*
-rwxr-xr-x. 1 root root 5.9M Oct 17 13:39 /usr/bin/qemu-aarch64-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-alpha-static*
-rwxr-xr-x. 1 root root 4.6M Oct 17 13:39 /usr/bin/qemu-armeb-static*
-rwxr-xr-x. 1 root root 4.6M Oct 17 13:39 /usr/bin/qemu-arm-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-cris-static*
-rwxr-xr-x. 1 root root 5.5M Oct 17 13:39 /usr/bin/qemu-hexagon-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-hppa-static*
-rwxr-xr-x. 1 root root 3.9M Oct 17 13:39 /usr/bin/qemu-i386-static*
-rwxr-xr-x. 1 root root 3.7M Oct 17 13:39 /usr/bin/qemu-m68k-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-microblazeel-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-microblaze-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mips64el-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mips64-static*
-rwxr-xr-x. 1 root root 4.1M Oct 17 13:39 /usr/bin/qemu-mipsel-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mipsn32el-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mipsn32-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-mips-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-nios2-static*
-rwxr-xr-x. 1 root root 3.4M Oct 17 13:39 /usr/bin/qemu-or1k-static*
-rwxr-xr-x. 1 root root 4.5M Oct 17 13:39 /usr/bin/qemu-ppc64le-static*
-rwxr-xr-x. 1 root root 4.5M Oct 17 13:39 /usr/bin/qemu-ppc64-static*
-rwxr-xr-x. 1 root root 4.4M Oct 17 13:39 /usr/bin/qemu-ppc-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-riscv32-static*
-rwxr-xr-x. 1 root root 4.2M Oct 17 13:39 /usr/bin/qemu-riscv64-static*
-rwxr-xr-x. 1 root root 3.8M Oct 17 13:39 /usr/bin/qemu-s390x-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-sh4eb-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-sh4-static*
-rwxr-xr-x. 1 root root 3.6M Oct 17 13:39 /usr/bin/qemu-sparc32plus-static*
-rwxr-xr-x. 1 root root 3.6M Oct 17 13:39 /usr/bin/qemu-sparc64-static*
-rwxr-xr-x. 1 root root 3.5M Oct 17 13:39 /usr/bin/qemu-sparc-static*
-rwxr-xr-x. 1 root root 3.9M Oct 17 13:39 /usr/bin/qemu-x86_64-static*
-rwxr-xr-x. 1 root root 6.4M Oct 17 13:39 /usr/bin/qemu-xtensaeb-static*
-rwxr-xr-x. 1 root root 6.4M Oct 17 13:39 /usr/bin/qemu-xtensa-static*
```

Reply:
Which proves Mirek is right, I guess.